### ISSUE TITLE
Copying the code

### ISSUE THREAD

**Issue by pushparajakln** [ArpitLokwani/ALNetworking#1]
Noticed that you have changed my name in all the files. I have updated network service with swift and nsurlsession that you cannot get from me 😄

**Comment by ArpitLokwani** [comment]
Yeah bro I have created new services in Swift too :) wait for my new release !!
